src/models/Settings.js

```diff
@@ -80,6 +81,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'features.hideSignOutLinkInMFA' : ['boolean', false, false],
       'features.customExpiredPassword': ['boolean', true, false],
       'features.registration': ['boolean', false, false],
+      'features.consent': ['boolean', false, false],
```

Comment:
how does feature flag has been passed into client side? Didnt find from this PR and suppose it's added before.

Reply:
will be passed by @larsjohansen-okta in loginpage_backbone.jsp